framework/po/page.js

```diff
@@ -0,0 +1,80 @@
+const AbstractComponent = require("./abstractComponent");
+const Component = require('./component');
+const Collection = require('./collection');
+
+class Page extends AbstractComponent {
+    constructor(locator, url = '/') {
+        super(locator);
+        this.pageUrl = url;
+    }
+    /**
+     * Returns webdriverio element by the element name from the page object 
+     * @param {String} elementPath - element name or full path to element
+     */
+    getElement(elementPath) {
+        let currentElement;
+        let currentComponent = this;
+        let that = this;
+        if (elementPath.includes(' ->')) {
```

Comment:
better to replace with regexp. May cause issues in case when user forgot to type spaces.